docs/reference/ocm_add_resource-configuration.md

```diff
@@ -313,6 +314,13 @@ with the field <code>type</code> in the <code>input</code> field:
     Basically, it is a good practice to use the component version for local resources
     This can be achieved by using templating for this attribute in the resource file.
   
+  - **<code>helmRepository</code>** *string*
+  
+    This OPTIONAL property can be set, if the helm chart should be loaded from 
+    a helm repository instead of the local filesystem. It describes
+    the base URL of the chart repository. If specified, the <code>path</code> field
+    must describe the name of the chart in the chart repository.
```

Comment:
I think it should also describe, that the version field is required when `helmRepository` is set.

Reply:
yes, I added this.